clients/python/example_check.py

```diff
@@ -0,0 +1,20 @@
+from .kh_client import *
```

Comment:
Since the `kh_client` is in the same directory you don't need the `.` -- this causes python to misinterpret the location of the import, creating a `ModuleNotFoundError`.